apollo/lib/schemas/user.py

```diff
@@ -42,28 +47,50 @@ class UserSchema(ORMBase):
     role: Optional[RoleSchema]
 
 
-class UpdateUserSchema(BaseModel):
-    old_password: constr(min_length=8, strip_whitespace=True)
-    password: constr(min_length=8, strip_whitespace=True)
-    password_confirm: constr(min_length=8, strip_whitespace=True)
+class UpdateUserSchema(BaseCreateOrUpdateUserSchema):
+    username: Optional[constr(
+        min_length=1, max_length=36, strip_whitespace=True)]
+    password: Optional[constr(min_length=8, strip_whitespace=True)]
+    password_confirm: Optional[constr(min_length=8, strip_whitespace=True)]
+    old_password: Optional[constr(min_length=8, strip_whitespace=True)]
 
-    @validator('password')
-    @classmethod
-    def no_whitespace_in_password(cls, value):
-        return check_for_whitespace(value)
+    class Config:
+        arbitrary_types_allowed = True
```

Comment:
Why do you want to allow arbitrary types? Disallowing them doesn't break the tests either.